src/NzbDrone.Core.Test/MetadataSource/SkyHook/SkyHookProxyFixture.cs

```diff
@@ -60,11 +63,35 @@ public void should_be_able_to_get_artist_detail(string mbId, string name)
 
             details.Item1.Name.Should().Be(name);
         }
+        
+        [TestCase("12fa3845-7c62-36e5-a8da-8be137155a72", null, "Hysteria")]
+        public void should_be_able_to_get_album_detail(string mbId, string release, string name)
+        {
+            var details = Subject.GetAlbumInfo(mbId, release);
+            
+            ValidateAlbums(new List<Album> {details.Item1});
+
+            details.Item1.Title.Should().Be(name);
+        }
+
+        [TestCase("12fa3845-7c62-36e5-a8da-8be137155a72", "3c186b52-ca73-46a3-a8e6-04559bfbb581",1, 13, "Hysteria")]
+        [TestCase("12fa3845-7c62-36e5-a8da-8be137155a72", "dee9ca6f-4f84-4359-82a9-b75a37ffc316",2, 27,"Hysteria")]
+        public void should_be_able_to_get_album_detail_with_release(string mbId, string release, int mediaCount, int trackCount, string name)
+        {
+            var details = Subject.GetAlbumInfo(mbId, release);
+
+            ValidateAlbums(new List<Album> { details.Item1 });
+
+            details.Item1.Media.Count.Should().Be(mediaCount);
+            details.Item2.Count.Should().Be(trackCount);
+
+            details.Item1.Title.Should().Be(name);
+        }
 
         [Test]
         public void getting_details_of_invalid_artist()
         {
-            Assert.Throws<BadRequestException>(() => Subject.GetArtistInfo("aaaaaa-aaa-aaaa-aaaa", 1));
```

Comment:
Don't know if we want to test this, but metadata should also return a 400 if the mbid is invalid. I suppose someone this could happen if someone inputs `lidarr:invalid` in the search, but there may be validation on lidarr's side that I don't know about.

Reply:
Currently on the search routes I have it checking for a valid GUID prior to sending the request to metadata. 

I'll add a quick test for a invalid guid response on the info requests. We are already catching the 400 in skyhook, so should be easy.